src/dotnet/APIView/APIViewWeb/Repositories/CosmosReviewRepository.cs

```diff
@@ -217,6 +217,7 @@ public async Task<IEnumerable<string>> GetReviewFirstLevelPropertiesAsync(string
                         queryStringBuilder.Append($" OR CONTAINS(r.Name, {query}, true)");
                         queryStringBuilder.Append($" OR CONTAINS(r.ServiceName, {query}, true)");
                         queryStringBuilder.Append($" OR CONTAINS(r.PackageDisplayName, {query}, true)");
+                        queryStringBuilder.Append($" OR ENDSWITH(ARRAY_SLICE(r.Revisions, -1)[0].Label, {query}, true)");
```

Comment:
I am a bit late to comment here. But CONTAINS would have been better here instead of Endswith to avoid any assumption that PR number will always be at the end of label. Label is generated by each language side script, and there is no specific rule for naming convention, and it can add any additional info in the future without knowing it will break in APIVIew side search.